pkg/time/rate/rate.go

```diff
@@ -124,9 +124,9 @@ func NewLimiter(r Limit, b int) *Limiter {
 	}
 }
 
```

Comment:
oh my bad for the replace, I'm not sure we want to change the time things in this package ...